epub33/rs/index.html

```diff
@@ -2067,92 +2073,193 @@ <h2>Processing structural semantics</h2>
-							data-cite="epub-33#sec-property-datatype"><code>property</code> data types</a> [[EPUB-33]].
-						When unprefixed terms are encountered in these attributes, the URL of the applicable default
-						vocabulary URL is used to <a href="#sec-property-datatype">expand the values</a>.</p>
-					<p>The default vocabulary URLs for these attributes are as follows:</p>
-					<ul data-cite="epub-33">
-						<li id="confreq-rs-pkg-meta-prefix">For the [^meta^] element's <a
-								data-cite="epub-33#attrdef-meta-property"><code>property</code> attribute</a>
-							[[epub-33]]: <br /><code>http://idpf.org/epub/vocab/package/meta/#</code></li>
-						<li id="confreq-rs-pkg-link-prefix">For the [^link^] element's <a
-								data-cite="epub-33#attrdef-link-rel"><code>rel</code></a> and <a
-								data-cite="epub-33#attrdef-properties"><code>properties</code></a> [[epub-33]]
-							attributes: <br /><code>http://idpf.org/epub/vocab/package/link/#</code></li>
-						<li id="confreq-rs-pkg-manifest-prefix">For the [^item^] element's <code>properties</code>
-							attribute: <br /><code>http://idpf.org/epub/vocab/package/item/#</code></li>
-						<li id="confreq-rs-spine-prefix">For the [^itemref^] element's <a
-								data-cite="epub-33#attrdef-properties"><code>properties</code> attribute</a>
-							[[epub-33]]: <br /><code>http://idpf.org/epub/vocab/package/itemref/#</code></li>
-					</ul>
-				</dd>
+			<p>This section defines an algorithm for obtaining an expanded URL from a <code>property</code> data type
+				value. It applies only to [=reading systems=] that support the [[epub-33]] vocabulary association
+				mechanisms.</p>
 
-				<dt id="sec-property-datatype">Expanding <code>property</code> data types</dt>
-				<dd>
-					<p>Reading systems MUST expand <a data-cite="epub-33#sec-property-datatype"><code>property</code>
-							values</a> [[epub-33]] as follows:</p>
-					<ul>
+			<p>Reading systems that do not support the [[epub-33]] vocabulary association mechanisms MAY process
+					<code>property</code> values as plain [=string=] values [[infra]]. It is not required to support the
+				expansion of these values to URLs to add reading system behaviors based on their presence.</p>
+
+			<p>The algorithm describes the process using the terminology and data types defined in [[infra]], and, if
+				successful, results in a [=string=] value with the expanded URL being returned. A <a
+					data-cite="infra#nulls">null</a> value is returned for invalid properties.</p>
+
+			<p>This algorithm takes the following arguments:</p>
+
+			<ul>
+				<li><var>value</var>: the <code>property</code> value to process.</li>
+				<li><var>attr</var>: the name of the attribute that contains <var>value</var>.</li>
+				<li><var>elem</var>: the name of the element that <var>attr</var> is attached to.</li>
+				<li><var>doc</var>: the document being processed (e.g., [=package document=], [=media overlay
+					document=], [=epub content document=]).</li>
```

Comment:
I am not sure why, but on my cloned version of the document these references do not appear as references, just plain texts. This may be some respec/github black art... :-( If it works on your side, ignore this.

Reply:
I'm getting proper links in my local version. Are all references broken for you?